website/content/en/highlights/2022-02-08-disk-buffer-v2-beta.md

```diff
@@ -0,0 +1,81 @@
+---
+date: "2022-02-08"
+title: "Beta release of Vector's new disk buffer implementation"
+description: "More consistent performance and resource usage for disk buffers"
+authors: ["tobz"]
+pr_numbers: [9476]
+release: "0.20.0"
+hide_on_release_notes: false
+badges:
+  type: enhancement
+---
+
+As the first major milestone in our work to improve the buffering support in Vector, we're excited
+to announce the beta release of a new disk buffer implementation, affectionately named "disk buffer
+v2".
+
+## Buffering in Vector
+
+In Vector, buffers serve the main purpose of temporarily absorbing spikes in load.  All sinks use
+some form of buffer between their inputs and the sink itself, and by default, an in-memory buffer is
+used.  While great for providing the lowest possible latency and highest throughput, in-memory
+buffers lack the ability to provide _data durability._ If any events are still in an in-memory
+buffer and Vector experiences an error that terminates the process, or the system running Vector
+itself crashes, those buffered events would be permanently lost.  As an alternative to in-memory
+buffers, we provide disk buffers, which allow writing the events to disk, providing durability and
+persistence, regardless of issues with Vector and the host system.
+
```

Comment:
I'd maybe add an example sink configuration here with buffers so that readers will more quickly recognize if they are using disk buffers or not.

Reply:
Done.